distribution/src/test/java/com/predic8/membrane/examples/env/ConsistentVersionNumbers.java

```diff
@@ -138,9 +138,12 @@ private static void transformRecursively(File baseDirectory, VersionTransformer
 	}
 
 	private static void handleConstants(File file, VersionTransformer versionTransformer) throws Exception {
-		//		String version = "5"; // fallback
-		Pattern versionPattern = Pattern.compile("(\\s*String version = \")(\\d+)(\";.*)");
-		handleByRegex(file, versionTransformer, versionPattern, v -> v.getMajor().toString());
+		//		String version = "5.1"; // fallback
+		handleByRegex(file, versionTransformer, getConstantsVersionPattern(), v -> "%d.%d".formatted(v.getMajor(), v.getMinor()));
+	}
+
+	static Pattern getConstantsVersionPattern() {
```

Comment:
Make pattern to a static field of this class instead of method.
Put Matching itself in a method and test against this method, do not test the pattern only.